testkit/src/main/resources/reference.conf

```diff
@@ -120,5 +119,31 @@ akka {
 
             event-stream=off
         }
+        default-dispatcher {
+            # The goal here is to reduce the number of threads spun up for test suites.
+            # Since every test suite currently
+            # has it's own actor system, we should limit the number of threads in the system
+            fork-join-executor {
+                # minimum of one thread for tests
+                parallelism-min = 1
+
+                # maximum of 2 threads for tests
+                parallelism-max = 2
+            }
+            default-blocking-io-dispatcher {
```

Comment:
It looks like the problem here is that this configuration is not defind for `internal-dispatcher`? 

```
    default-blocking-io-dispatcher {
      type = "Dispatcher"
      executor = "thread-pool-executor"
      throughput = 1
      thread-pool-executor {
        fixed-pool-size = 16
      }
    }
```

We probably need to use the `fixed-pool-size` config

https://doc.akka.io//docs/akka/current/general/configuration-reference.html